src/app/models/Cake.ts

```diff
@@ -0,0 +1,35 @@
+const enum CakeStatus {
+  LastUnits = "LastUnits",
+  OutOfStock = "OutOfStock",
+  Available = "Available",
+}
+
+interface ICake {
```

Comment:
Since we are using a Single File Responsibility file distribution, this Interface should live in a separate file named `cake.interface.ts`.

Reply:
Makes sense